cmd/set/theme/theme.go

```diff
@@ -0,0 +1,246 @@
+	}
+
+	err = hostedPageCalls()
+	if err != nil {
+		return err
+	}
+
+	err = otherCalls()
+	if err != nil {
+		return err
+	}
+
+	err = updateCalls()
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("Your theme has been changed")
+
+	return nil
+}
+
+func contains(s []string, str string) bool {
+	for _, v := range s {
+		if v == str {
+			return true
+		}
+	}
+	return false
+}
+
+func getTheme() (string, error) {
+	resp, err := api.GetPage()
+	if err != nil {
+		return "", err
+	}
+	themeMap := map[string]string{
+		"1": "London",
+		"2": "Tokyo",
+		"3": "Helsinki",
+	}
+	index := resp.Pages[0].Status
+	return themeMap[index], nil
+}
+
+func resetCalls() error {
+	conf := config.GetInstance()
+	reset := conf.AdminConsoleAPIDomain + "/deployment/hostedPage/reset"
+	bodyAuth, _ := json.Marshal(map[string]string{
+		"pageType": "Auth",
+	})
+	_, err := request.Rest(http.MethodPut, reset, nil, string(bodyAuth))
+	if err != nil {
+		return err
+	}
+	bodyProfile, _ := json.Marshal(map[string]string{
+		"pageType": "Profile",
+	})
+	_, err = request.Rest(http.MethodPut, reset, nil, string(bodyProfile))
+	if err != nil {
+		return err
+	}
+
+	return nil
+
+}
+
+func updateCalls() error {
+	conf := config.GetInstance()
+	update := conf.AdminConsoleAPIDomain + "/deployment/hostedPage/update"
+	var customJSAuth body
+	var customJSProf body
+	cjs := CustomJS{
+		Content:  "options = {\n    \"language\": \"English\"\n}",
+		fileName: "lr-interface-options",
+	}
+	customJSAuth.CustomJS = append(customJSAuth.CustomJS, cjs)
+	customJSProf.CustomJS = append(customJSProf.CustomJS, cjs)
+
+	customJSAuth.PageType = "Auth"
+	body1, _ := json.Marshal(customJSAuth)
+	_, err := request.Rest(http.MethodPost, update, nil, string(body1))
+	if err != nil {
+		return err
+	}
+	customJSProf.PageType = "Profile"
+	body2, _ := json.Marshal(customJSProf)
+	_, err = request.Rest(http.MethodPost, update, nil, string(body2))
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func otherCalls() error {
+	conf := config.GetInstance()
+	profile := conf.AdminConsoleAPIDomain + "/deployment/hostedPage/script/Profile"
+	bodyProfile, _ := json.Marshal(map[string]string{
+		"url": "https://hosted-pages.lrinternal.com/Themes/profile/html/profile.html",
+	})
+	_, err := request.Rest(http.MethodPost, profile, nil, string(bodyProfile))
+	if err != nil {
+		return err
+	}
+
+	auth := conf.AdminConsoleAPIDomain + "/deployment/hostedPage/script/Auth"
+	bodyAuth, _ := json.Marshal(map[string]string{
+		"url": "https://hosted-pages.lrcontent.com/lrcli/lr-interface-options.js",
+	})
+	_, err = request.Rest(http.MethodPost, auth, nil, string(bodyAuth))
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func hostedPageCalls() error {
+	conf := config.GetInstance()
+	hosted := conf.AdminConsoleAPIDomain + "/deployment/hostedpage"
+	auths := map[string]cmdutil.ThemeType{
+		"London":   cmdutil.Theme1Auth,
+		"Tokyo":    cmdutil.Theme2Auth,
+		"Helsinki": cmdutil.Theme3Auth,
+	}
```

Comment:
We can create one function in the `constant.go` which will have this map, and it takes the argument as an theme name and based on that it should return only that particular theme object.